src/arrayMethodSort.js

```diff
@@ -4,9 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]];
+        }
+      }
+    }
```

Comment:
While bubble sort is a valid approach, it's worth noting that it is not the most efficient sorting algorithm, especially for large arrays. It has a time complexity of O(n^2) which can be quite slow. If performance is a concern, consider implementing a more efficient algorithm like quicksort or mergesort.